Lab_5.ipynb

```diff
@@ -0,0 +1,1121 @@
+       "0       3.244343    0.006256  0.961697  0.598986  0.425024  0.622843   \n",
+       "1       6.481473    0.000155  0.896003  0.633385  0.476566  0.706900   \n",
+       "2       2.634788    0.005682  0.861678  0.486505  0.264631  0.564602   \n",
+       "3      13.389845    0.000432  0.952917  0.856141  0.730438  0.898442   \n",
+       "4      11.342156    0.002575  0.868770  0.645362  0.484853  0.742845   \n",
+       "...          ...         ...       ...       ...       ...       ...   \n",
+       "99995   4.368135    0.000900  0.671057  0.294523  0.255893  0.438894   \n",
+       "99996   6.749812    0.002829  0.718651  0.293553  0.268076  0.408478   \n",
+       "99997   2.712970    0.002456  0.859900  0.642900  0.456476  0.747645   \n",
+       "99998   1.972101    0.005786  0.930685  0.672938  0.361115  0.723057   \n",
+       "99999  17.887665    0.005031  1.137163  0.926213  0.904575  0.814495   \n",
+       "\n",
+       "            t32  KtDeltaR  \n",
+       "0      0.709573  0.082995  \n",
+       "1      0.752412  0.439346  \n",
+       "2      0.543942  0.251453  \n",
+       "3      0.853175  0.121666  \n",
+       "4      0.751289  0.386788  \n",
+       "...         ...       ...  \n",
+       "99995  0.868838  0.282015  \n",
+       "99996  0.913209  0.261662  \n",
+       "99997  0.710027  0.147913  \n",
+       "99998  0.536624  0.567817  \n",
+       "99999  0.976638  0.603431  \n",
+       "\n",
+       "[100000 rows x 14 columns]"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "d = {keys[0] : background_dict[keys[0]]}\n",
+    "for i in range(len(keys)):\n",
+    "    d[keys[i]] = background_dict[keys[i]]\n",
+    "\n",
+    "df = pd.DataFrame(data=d)\n",
+    "df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6c8ad225-ce3e-4c9a-b352-3dd9f4e76ab9",
+   "metadata": {},
+   "source": [
+    "### Terminology: \n",
+    "- pt: Transverse momentum\n",
+    "- eta: Pseudorapidity\n",
+    "- phi: Azimuthal angle\n",
+    "- mass\n",
+    "- ee2: energy correlation function\n",
+    "- ee3: energy correlation function\n",
+    "- d2: energy correlation function\n",
+    "- angularity\n",
+    "- t1: subjettiness with one particle\n",
+    "- t2: subjettiness with 2 particles\n",
+    "- t3: subjettiness with 3 particles\n",
+    "- t21: t2 / t1\n",
+    "- t32: t3 / t2\n",
+    "- KtDeltaR: Delta R of 2 subjets within the large R jet"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a1dc63ab-c2f8-4a15-afcc-d4196b4d91b1",
+   "metadata": {},
+   "source": [
+    "**Do all of the features provide discrimination power between signal and background?**\n",
+    "\n",
+    "Generally yes, as seen by the plots below. Phi does not, as expected (ranges from $-\\pi$ to $\\pi$ for both) and t32 has significant overlap between signal and background."
```

Comment:
Is this just qualitative or is there some numerical reasoning you're using here? If qualitative may want to mention that